src/grading-settings/grading-scale/GradingScale.scss

```diff
@@ -9,6 +9,7 @@
     height: 3.125rem;
     width: 100%;
     border: 1px solid $black;
+    overflow: hidden;
```

Comment:
This is added because the text elements in the segment can overflow and interfere with the add segment button.